style.css

```diff
@@ -0,0 +1,594 @@
+}
+
+.sub-title {
+  line-height: 1.563rem;
+  font-size: 30px;
+  padding: 2%;
+  color: orangered;
+}
+
+.head-title {
+  color: white;
+  font-weight: bold;
+  font-size: 70px;
+  background-image: url(./assets/orangeImg.jpg);
+  letter-spacing: 5px;
+}
+
+.venue {
+  padding-block: 1%;
+  color: rgb(123, 110, 110);
+}
+
+.date {
+  padding-block: 3%;
+  color: rgb(209, 161, 161);
+}
+
+.second-section {
+  color: #fff;
+  background-color: #2a4073;
+  padding: 2%;
+}
+
+.second-section .inner-div {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.hr {
+  width: 10%;
+  border-color: red;
+  margin-inline: auto;
+  margin-top: 1%;
+}
+
+.content-list {
+  color: rgb(225, 202, 196);
+  font-size: 1.562rem;
+}
+
+.inner-div .content-list {
+  padding: 5%;
+}
+
+.program {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  background-color: #fff1;
+  padding: 2%;
+}
+
+.program-content {
+  list-style: none;
+  display: flex;
+  padding-block: 3%;
+  align-items: center;
+  text-align: center;
+  gap: 0.5rem;
+}
+
+.icon {
+  font-size: 1em;
+}
+
+.description {
+  font-size: 14px;
+  padding: 2%;
+}
+
+.details {
+  color: #fff;
+  font-size: 0.875rem;
+  padding: 2.5rem 0;
+}
+
+.btn {
+  display: none;
+}
+
+.speakers {
+  padding: 2%;
+}
+
+.speakersList {
+  list-style: none;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 45px;
+  padding: 5%;
+}
+
+.speakers .inner-div {
+  align-items: center;
+  flex-direction: column;
+  display: flex;
+}
+
+.speaker {
+  display: flex;
+  gap: 10px;
+}
+
+.backFrame {
+  display: flex;
+  background-image: url(./assets/bgFrame.png);
+  background-size: 60% 60%;
+  background-repeat: no-repeat;
+  justify-content: end;
+  align-items: flex-end;
+  width: 90%;
+}
+
+.speaker-detail {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.speakerImg {
+  width: 92%;
+  height: 92%;
+}
+
+.speaker-name h3 {
+  float: left;
+  padding: 2%;
+}
+
+.speaker-affiliation h4 {
+  float: left;
+  color: rgb(255, 55, 0);
+  font-style: italic;
+}
+
+.speaker-description p {
+  padding-block: 3%;
+}
+
+.partners {
+  background-color: #272a31;
+  color: #fff;
+}
+
+.partners-List {
+  list-style: none;
+  display: flex;
+  gap: 6rem;
+  padding-inline: 5%;
+  padding-block: 4%;
+}
+
+.partners .inner-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+footer {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 3%;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.partners-List-Logo {
+  width: fit-content;
+  height: 2rem;
+  object-fit: contain;
+}
+
+.email {
+  font-size: 1.3rem;
+}
+
+.logoContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}

```

Comment:
- [ ] Please make sure your design matches the given design template as much as possible as specified in the project requirement. 
<img width="1088" alt="Screenshot 2023-06-01 at 11 15 26" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/e71f015f-4e87-4783-94cc-aaa3bd2a66da">
 <table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="873" alt="Screenshot 2023-06-01 at 11 38 32" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/749d990e-38ac-49f9-a6e6-64cd77fb3a19">
</td>
<td>
<img width="1438" alt="Screenshot 2023-06-01 at 11 44 17" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/febc35d3-4ce4-4a11-8557-f5c20b9c7c75">
</td>
</tr>
</table>